### ISSUE TITLE
DCQL: `namespace` and `claim_name` vs `path` 

### ISSUE THREAD

**Issue by bc-pi** [openid/OpenID4VP#293]
> I still don't understand why these can't or shouldn't or aren't conveyed with `path` too i.e. `["org.iso.18013.5.1","first_name"]` but I'll not stand in the way of progress for this PR and make a note to myself to ask/complain about it later. Or maybe just forget.

_Originally posted by @bc-pi in https://github.com/openid/OpenID4VP/pull/266#discussion_r1811480510_

**Comment by markuskreusch** [comment]
I am slightly in favor of using path as the sole option to reference data inside of a credential. This will work for data of any structure, if deeply nested and complex or more simple like for mdoc where it is only a list of claims next to each other.

The semantics of how path is used would be credential format specific in this case, so the spec would define claims path pointer for JSON based credentials and a two element path array referencing namespace and claim name for mdoc 
credentials. Additional credential formats would reuse path as data referencing mechanism if possible.

Having only path will keep the data objects and parsing logic more simple and defer credential specific stuff to the path evaluation logic.

**Comment by awoie** [comment]
I'm against semantic overloading of parameters depending on the credential format. That is why I'm in favor of https://github.com/openid/OpenID4VP/issues/284#issuecomment-2430108399.

**Comment by bc-pi** [comment]
From https://github.com/openid/OpenID4VP/pull/266#discussion_r1814171604 @leecam said:

> yeah, I still don't really like this special handling for mdocs. Its pain when implementing it too. So I guess same question from me, can everything just be paths?

**Comment by Sakurann** [comment]
WG discussion
also taking into account implementation experience, rough agreement to use path for both mdoc and json-based credential formats (like sd-jwt vc).

**Comment by jogu** [comment]
This didn't make it into VP ID3 right? I've moved it to the 'final' milestone instead.

**Comment by danielfett** [comment]
No, this is the PR: https://github.com/openid/OpenID4VP/pull/368/files

**Comment by c2bo** [comment]
Can be closed as the PR (#368) got merged, correct?

**Comment by Sakurann** [comment]
resolved with #368
